biblio/how_to_cite_gudhi.bib.in

```diff
@@ -127,42 +157,166 @@
 , year =        @GUDHI_VERSION_YEAR@
 }
 
-@incollection{gudhi:cython
+@incollection{gudhi:PersistenceRepresentations
+, author =  "Pawel Dlotko"
+, title =   "Persistence representations"
+, publisher =  "{GUDHI Editorial Board}"
+, edition =     "{@GUDHI_VERSION@}"
+, booktitle =   "{GUDHI} User and Reference Manual"
+, url = "https://gudhi.inria.fr/doc/@GUDHI_VERSION@/group___persistence__representations.html"
+, year =        @GUDHI_VERSION_YEAR@
+}
+
+@incollection{gudhi:SubSampling
+, author =  "Cl\'ement Jamin and Siargey Kachanovich"
+, title =   "Subsampling"
+, publisher =  "{GUDHI Editorial Board}"
+, edition =     "{@GUDHI_VERSION@}"
+, booktitle =   "{GUDHI} User and Reference Manual"
+, url = "https://gudhi.inria.fr/doc/@GUDHI_VERSION@/group__subsampling.html"
+, year =        @GUDHI_VERSION_YEAR@
+}
+
+@incollection{gudhi:SpatialSearching
+, author =  "Cl\'ement Jamin"
+, title =   "Spatial searching"
+, publisher =  "{GUDHI Editorial Board}"
+, edition =     "{@GUDHI_VERSION@}"
+, booktitle =   "{GUDHI} User and Reference Manual"
+, url = "https://gudhi.inria.fr/doc/@GUDHI_VERSION@/group__spatial__searching.html"
+, year =        @GUDHI_VERSION_YEAR@
+}
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+% Python specific gudhi modules
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+
+@incollection{gudhi:CubicalPersistenceTensorFlowLayer
+, author =  "Mathieu Carri\`ere"
+, title =   "TensorFlow layer for cubical persistence"
+, publisher =  "{GUDHI Editorial Board}"
+, edition =     "{@GUDHI_VERSION@}"
+, booktitle =   "{GUDHI} User and Reference Manual"
+, url = "https://gudhi.inria.fr/python/@GUDHI_VERSION@/cubical_complex_tflow_itf_ref.html"
+, year =        @GUDHI_VERSION_YEAR@
+}
+
+@incollection{gudhi:CubicalPersistenceScikitlearnInterface
 , author =  "Vincent Rouvreau"
-, title =   "Cython interface"
+, title =   "Cubical complex persistence scikit-learn like interface"
 , publisher =  "{GUDHI Editorial Board}"
 , edition =     "{@GUDHI_VERSION@}"
 , booktitle =   "{GUDHI} User and Reference Manual"
-, url = "https://gudhi.inria.fr/python/@GUDHI_VERSION@/"
+, url = "https://gudhi.inria.fr/python/@GUDHI_VERSION@/cubical_complex_sklearn_itf_ref.html"
 , year =        @GUDHI_VERSION_YEAR@
 }
 
-@incollection{gudhi:CoverComplex
+@incollection{gudhi:LowerStarPersistenceTensorFlowLayer
 , author =  "Mathieu Carri\`ere"
-, title =   "Cover complex"
+, title =   "TensorFlow layer for lower-star persistence on simplex trees"
 , publisher =  "{GUDHI Editorial Board}"
 , edition =     "{@GUDHI_VERSION@}"
 , booktitle =   "{GUDHI} User and Reference Manual"
-, url = "https://gudhi.inria.fr/doc/@GUDHI_VERSION@/group__cover__complex.html"
+, url = "https://gudhi.inria.fr/python/@GUDHI_VERSION@/ls_simplex_tree_tflow_itf_ref.html"
 , year =        @GUDHI_VERSION_YEAR@
 }
 
-@incollection{gudhi:PersistenceRepresentations
-, author =  "Pawel Dlotko"
-, title =   "Persistence representations"
+@incollection{gudhi:WeightedRipsComplex
+, author =  "Yuichi Ike"
```

Comment:
I see more authors in the file.

Reply:
Fixed on [8a7353e](https://github.com/GUDHI/gudhi-devel/pull/700/commits/8a7353e0f98f03ce98499bb5d5648193fd95df1c)